src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -45,6 +44,8 @@ public class ErrorListService
     {
         private const string VersionRegexPattern = @"""version""\s*:\s*""(?<version>[\d.]+)""";
 
+        private const string Sha256HaskKey = "sha-256";
```

Comment:
>Sha256HaskKey [](start = 29, length = 13)

HashKey

Reply:
fixed

---
In reply to: [601272802](https://github.com/microsoft/sarif-visualstudio-extension/pull/384#discussion_r601272802) [](ancestors = 601272802)